src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -0,0 +1,28 @@
+package com.kodilla.ecommercee.domain;
+import lombok.*;
+
+import javax.persistence.*;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue
+    @NonNull
+    private Long id;
+
+    @NonNull
+    @Column(unique = true)
+    private String userName;
```

Comment:
@Column(unique = true, nullable = false) - będzie krócej.
https://thoughts-on-java.org/hibernate-tips-whats-the-difference-between-column-nullable-false-and-notnull/

Reply:
Hmm.. Z artykułu, który podlinkowałeś, wynika, że wskazane jest używanie adnotacji `@NotNull` zamiast atrybutu `nullable = false` :D